nbs/src/tbats.ipynb

```diff
@@ -26,11 +26,11 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.            x_nought = (x_nought_scaled/sigma)+mu 
</pre><p>This should be <code>x_nought = x_nought * sigma + mu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>of course, since the original transformation is z = (x-mu)/sigma</p>